diagnostic.md

```diff
@@ -6,70 +6,70 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+// find and update
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+// the controller
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+// bundle exec rake will update the gem files and bd:migrate will migrate a database
```

Comment:
`bundle exec rake db:migrate` executes all of the database migration files in `db/migrate` and generates a new database schema file, `db/schema.rb`.